### ISSUE TITLE
save() Callback Returning Record Inconsistent with Database

### ISSUE THREAD

**Issue by itsalaidbacklife** [balderdashy/sails-docs#514]
I have a couple actions with nested save() calls, where several of the records returned to the callbacks passed to their respective save() calls have the expected values, but the actual database does not reflect the new changes. Here is my [stack overflow.](http://stackoverflow.com/questions/31347281/discrepancy-between-data-in-save-callback-and-database)

I realize that the problem may be with the way I am attempting to nest several saves inside of each other, but I want to have all the data returned by the database in order to send it to my clients with a publishUpdate(). Is that fundamentally unreasonable?

I apologize if this isn't the place to post this problem!

**Comment by sailsbot** [comment]
Thanks for posting, @itsalaidbacklife.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
